config/install/core.entity_view_display.node.stanford_person.default.yml

```diff
@@ -766,6 +766,3 @@ content:
 hidden:
   layout_builder__layout: true
   su_person_research_interests: true
-  su_person_academic_appt: true
-  su_person_admin_appts: true
-  su_person_scholarly_interests: true
```

Comment:
These should probably stay.

Reply:
Ok. What is the best way to get them back? Do I add the lines manually back?

Reply:
That would be the easy way. Sure.

Reply:
👍 Ok. I did it that way